client/verta/requirements.txt

```
@@ -7,3 +7,4 @@ pytimeparse >= 1.1.8, < 2.0
 pyyaml >= 5.1, < 7.0
 requests >= 2.25, < 3.0
 urllib3 >= 1.26.0
+jsonschema>=4.17.3
```

Also, please insert this in alphabetical order!

```suggestion
jsonschema >= 4.17.3
```